### PR TITLE
[DOC] Fixed minor typos, grammar errors and moved imports up in all examples

### PR DESCRIPTION
**Pull request by ShrishtiHore**
This PR is based on the previous PR's and discussions we had. I have moved the imports of all examples at the top of the code. I also tried fixing as many typos as I could. 

It would be great if @skoudoro and @arokem please review it and give your feedback. 

There are 4 files I didn't make any changes to because they were format(pattern) based examples with multiple samples in one source code. They are tracking_introduction.py,  workflow_creation.py, segment_clustering_feature.py and segment_clustering_metric.py

Please check it out and tell any corrections needed.

### CODE REVIEW

File: doc/examples/brain_extraction_dwi.py [hunk 1]
```diff
@@ -22,6 +23,12 @@
 models. For this example, the data comes from a 1.5 Tesla Siemens MRI.
 """
 
+
+from dipy.data import get_fnames
+from dipy.io.image import load_nifti, save_nifti
+from dipy.segment.mask import median_otsu
```
Comment (skoudoro) [inline doc/examples/brain_extraction_dwi.py:29]: the block above appears twice and is duplicated with the import on the top

File: doc/examples/brain_extraction_dwi.py [hunk 2]
```diff
@@ -22,6 +23,12 @@
...
+from dipy.core.histeq import histeq
```
Comment (skoudoro) [inline doc/examples/brain_extraction_dwi.py:30]: Why this one is not on the top ?

File: doc/examples/combined_workflow_creation.py [hunk 3]
```diff
@@ -108,7 +109,6 @@ def run(self, input_files, out_dir='', out_file='processed.nii.gz'):
 file located in ``bin``.
 """
 
-from dipy.workflows.flow_runner import run_flow
```
[the remark this answers is not in the document]
Comment (ShrishtiHore) [reply doc/examples/combined_workflow_creation.py:111]: okay thanks a lot.

File: doc/examples/brain_extraction_dwi.py (hunk 1 above)
Comment (ShrishtiHore) [reply doc/examples/brain_extraction_dwi.py:29]: okay got it.

File: doc/examples/brain_extraction_dwi.py (hunk 2 above)
Comment (ShrishtiHore) [reply doc/examples/brain_extraction_dwi.py:30]: Sorry made a mistake in there.

**Comment by ShrishtiHore** [comment]
As @skoudoro suggested I am closing this PR and I will review the https://github.com/dipy/dipy/pull/1990 PR and make some changes in same branches that he is working on. Thank You for your reply @skoudoro

**Pull request closed without merging**